### DISCUSSION TITLE
Why don't my instances join raids?

### DISCUSSION THREAD

**Discussion by deleeuwpeter** [KevinBytesTheDust/CVAmp#202] [General]
I suspect the (already viewing) bots aren't joining a raid when the streamer initiates it. The raid viewer number doesn't reflect the amount of bots; it's much less.

Is this normal behavior or am I doing something wrong?

**Comment by jlplenio** [comment]
From what I have been told by our awesome community, this is normal behavior. Only logged-in users follow raids. We are testing a version for logging in and chatting over at [ko-fi](https://ko-fi.com/jlplenio/posts).
